Packs/LinuxEventsCollection/README.md

```diff
@@ -1,52 +1,75 @@
 # Linux Events Collection
-This pack includes XSIAM content.
 
-## Collect Events from Vendor
+## Collector Configuration
 
-In order to use the collector, you can use one of the following options to collect events from the vendor:
+You could use any one of the following options to collect Linux events into Cortex XSIAM:
```

Comment:
Recommend:
Use one of the following options to collect Linux events into Cortex XSIAM:

Reply:
fixed: 
https://github.com/demisto/content/pull/36882/files#diff-547b4347d8e4a399e431acab3ca503579d3d23c0f8854847ee85c97343769646R5